src/llmcompressor/utils/metric_logging.py

```diff
@@ -51,3 +53,50 @@ def get_layer_size_mb(module: Module) -> float:
     total_size_mb = total_size / (1e6)  # Convert bytes to MB
 
     return total_size_mb
+
+
+class CompressionLogger:
```

Comment:
Why is this required given the current logging that already exists in gptq?

Reply:
Breaking this out means more real estate is dedicated to describing the algorithm, not logging